Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml

```diff
@@ -13,6 +13,7 @@ query: |
   IdentityInfo
   | where AccountCreationTime < ago(7d)
   | where GroupMembership contains adminGroup
+  | extend Account_0_AadUserId = AccountObjectId
 entityMappings:
   - entityType: Account
     fieldMappings:
```

Comment:
can we add versions to all Hunting queries? version: 2.0.0

Reply:
done